src/main/java/org/spongepowered/api/item/ItemType.java

```diff
@@ -44,8 +43,8 @@
     String getId();
 
     /**
-     * Get the default maximum quantity for {@link ItemStack}s
-     * of this item.
+     * Get the default maximum quantity for
+     * {@link org.spongepowered.api.item.inventory.ItemStack}s of this item.
```

Comment:
Fully qualifying doesn't look or read as well compared to using a shorthand w/ import